### ISSUE TITLE
Swift logo in header needs alt-text

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-design-system#729]
Currently, when ```Swift and tickets``` is used, the SVG of the Swift logo does not have alt text.

Therefore, to screen readers it reads as ```and tickets```.

We need to add alt-text to say ```Swift```
